### PR TITLE
feat: clear separation of content and layout components

### PR DESCRIPTION
**Pull request by jfcurran**
The purpose of this PR is to 
- simplify the structure of our website
- clearly separate text and image content from functional layout components
- purge unused code and reduce debt

Content found on pages will now be found in the directory `/page-content`. For instance, content found viewing index will now be found in `/page-content/content-index.tsx`. additionally, shared contents like 
- stats 
- quotes 
- adopters
- open source product descriptions
- error messages
- banners
- info regarding the team (eg about.tsx)
will be handled in `/page-content/shared`.
Furthermore, information regarding product pricing is handled and thus isolated in `/page-content/pricing-tiers` 

Unused components and files have long been bloating the repo and I have made sure to remove most after careful deliberations. If anything flew under my radar please be sure to let me know, as external usage of files from this repo are hard to keep track of.

### CODE REVIEW

**Comment by jfcurran** [comment]
@vinckr Please carefully review the new page-contents - i have made sure to remove typos as i went along, however four eyes are infinitely better than two! ᵃᶜᵗᵘᵃˡˡʸ ᵒⁿˡʸ ᵇʸ ᵃ ᶠᵃᶜᵗᵒʳ ᵒᶠ ²

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ contact_links:
     about: Please ask and answer questions here, show your implementations and discuss ideas.
   - name: Ory Chat
     url: https://www.ory.sh/chat
-    about: Hang out with other Ory community members and ask and answer questions.
+    about: Hang out with other Ory community members to ask and answer questions.
```
Comment (aeneasr) [inline .github/ISSUE_TEMPLATE/config.yml:8]: Please fix this in ory/meta

**Review by aeneasr — COMMENTED** [review]
Unfortunately very difficult to review so if you have checked that everything works on every page it's good from my side

**Comment by aeneasr** [comment]
Once @vinckr approves we can merge

**Comment by vinckr** [comment]
I will take a look now!

**Review by vinckr — APPROVED** [review]
A lot of changes 😮 

I looked into all the content pages and everything looks as it should. 
Also tested briefly on an Ipad & Android phone, all checks out.

**Pull request merged**